src/routes/donations.js

```diff
@@ -60,16 +61,24 @@ router.post("/dashboard", function(req, res, next) {
 
   db.Donation.findAll({
     attributes: [
-      'sourceId',
+      [Sequelize.fn('', Sequelize.col('Source.description')), 'sourceDescription'],
       [Sequelize.fn('SUM', Sequelize.col('donationAmount')), 'totalAmountDonated']
```

Comment:
@clukhei I believe you're referring to lines 64 and 65 as there being 2 functions? Is it a problem?